build.sbt

```diff
@@ -93,10 +93,12 @@ lazy val noPublish = List(
 )
 
 lazy val benchmarks = project
-  .dependsOn(metals)
+  .disablePlugins(ScriptedPlugin)
```

Comment:
are all these `disablePlugins` necessary even after https://github.com/sbt/sbt/pull/3875?

Reply:
I'm not sure. I saw this failure: https://travis-ci.org/laughedelic/metals/jobs/363624534#L931

and then I also saw this commit in bloop: https://github.com/scalacenter/bloop/commit/7b45b9b8120867a258520f0aba42f6ade87471d1

Reply:
Oh, and https://github.com/sbt/sbt/pull/3875 isn't in sbt 1.1 yet, it's targeted to 1.2.